src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+
+    public Square(String color, double side) {
```

Comment:
The constructor should not directly accept a 'side' value. Instead, consider creating a separate variable before passing it to the constructor for better readability.